### DISCUSSION TITLE
vesc 6.05 variable fixes

### DISCUSSION THREAD

**Discussion by aka13-404** [aka13-404/VSETT-LISP#5] [Announcements]
This release does not bring new features.
With this version, the problems with variable declaration in latest 6.05 beta should be fixed and working properly now.
Please let me know if this works on your setup. 

<hr /><em>This discussion was created from the release <a href='https://github.com/aka13-404/VSETT-LISP/releases/tag/v1.0.1'>vesc 6.05 variable fixes</a>.</em>

**Comment by Will-i-Code** [comment]
Could you share a bit more on your Makerbase 84100HP Dual Setup?
How display you connect to turn on/off both controllers via the power button on display?

Single Controller setup looks straightforward enough thanks to your instructions.  Thanks

**Reply by aka13-404** [reply]
It's the same, just connect them afterwards in parallel. They share the same ground either way, so nothing to worry about.
